packages/lib-react-components/src/Markdownz/Markdownz.js

```diff
@@ -75,6 +106,8 @@ class Markdownz extends React.Component {
       console.warn('Overriding the rendering function for the img tag may break the syntax support for image resizing and using image markup for video and audio. Are you sure you want to do this?')
     }
 
+    const childrenWIthImageResize = this.findResizedImages(children)
```

Comment:
typo
```suggestion
    const childrenWithImageResize = this.findResizedImages(children)
```

Reply:
Does the regex need to be run on every render? It seems like it's only needed when the component first renders and, optionally, if the `children` string changes.

Reply:
Agreed! I'd like to save this for a future PR though.

Reply:
Yes! If `remark()` is anything like `markdown-it` then setting it up from scratch on every render might be expensive too. Definitely outside the scope of this PR.